tools/serverpod_cli/lib/src/analyzer/models/definitions.dart

```diff
@@ -85,6 +85,14 @@ class ClassDefinition extends SerializableModelDefinition {
         .cast()
         .firstWhere((element) => element.name == name, orElse: () => null);
   }
+
+  ClassDefinition? get parentClass =>
+      extendsClass is ResolvedInheritanceDefinition
+          ? (extendsClass as ResolvedInheritanceDefinition).classDefinition
+          : null;
```

Comment:
Nit: Do not use arrow functions for multiline statements, opt for a normal method in these cases.

Suggestion: Do a local type check of the variable here instead so you can avoid doing a cast. The code works but when casting you are opening you self up to cast errors if someone modifies the code in the wrong way.

Instead this will solve the problem in a cleaner way:

```dart
ClassDefinition? get parentClass {
   var extendedClass = extendsClass;
   if (extendedClass is! ResolvedInheritanceDefinition) return null;

   return extendedClass.classDefinition;
}
```